views/scss/result.scss

```diff
@@ -164,3 +164,10 @@ tr td {
     text-shadow: none;
     color: #3e7da7;
 }
+
+.btn-secondary {
```

Comment:
In order to prevent having to use '!important' could we add more specificity to the css target?

i.e: 
```scss
.btn-info.btn-secondary {
    &.preview, &.result-filter-btn {
    ....
    }
}
```

Reply:
Excellent! Maybe when SCSS compile, generate more code but since the root of problem is divided styles across platform, this is better workaround for now